internal/datasource/nginx/process.go

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
can you add a test for this file? At least  defaultToNginxCommandForProcessPath and sanitizeExeDeletedPath functions